onnx/checker.py

```diff
@@ -80,36 +85,39 @@ def check_tensor(tensor: TensorProto, ctx: C.CheckerContext = DEFAULT_CONTEXT) -
 
 
 def check_attribute(
-    attr: AttributeProto, ctx: C.CheckerContext = DEFAULT_CONTEXT
+    attr: AttributeProto,
+    ctx: C.CheckerContext = DEFAULT_CONTEXT,
+    lex_ctx: C.LexicalScopeContext = LEXICAL_SCOPE_CONTEXT,
 ) -> None:
     _ensure_proto_type(attr, AttributeProto)
-    return C.check_attribute(attr.SerializeToString(), ctx)
+    return C.check_attribute(attr.SerializeToString(), ctx, lex_ctx)
 
 
-def check_node(node: NodeProto, ctx: C.CheckerContext = DEFAULT_CONTEXT) -> None:
+def check_node(
+    node: NodeProto,
+    ctx: C.CheckerContext = DEFAULT_CONTEXT,
+    lex_ctx: C.LexicalScopeContext = LEXICAL_SCOPE_CONTEXT,
+) -> None:
     _ensure_proto_type(node, NodeProto)
-    return C.check_node(node.SerializeToString(), ctx)
+    return C.check_node(node.SerializeToString(), ctx, lex_ctx)
 
 
 def check_function(
-    function: FunctionProto, ctx: C.CheckerContext | None = None
```

Comment:
@jcwchen I removed the Optional type for both `CheckerContext` and `LexicalScopeContext` here